_docs/concepts/network-and-auth/auth.md

```diff
@@ -128,3 +128,27 @@ in [Mixer]({{home}}/docs/concepts/policy-and-control/mixer.html) such that photo
 
 In this setup, Istio CA is able to provide keys and certificates management for all namespaces, and isolate microservice deployments from each other.
 
+## Future work
+
+*   Fine-grained authorization and auditing
+
+*   Key and certificate rotation and revocation
```

Comment:
remove this one, since key rotation is already supported, and we may not need revocation with short lived cert